internal/commands/import_test.go

```diff
@@ -10,15 +10,20 @@ import (
 )
 
 func TestImport_ImportSarifFileWithCorrectFlags_CreateImportSuccessfully(t *testing.T) {
```

Comment:
same here for project/ I may misunderstand the current functionality and the new